index.js

```diff
@@ -280,16 +285,39 @@ function parseDnsOverHttpsRecord (datDns, name, body, dnsTxtRegex) {
   return res
 }
 
-function fetchWellKnownRecord (name, recordName) {
-  return new Promise((resolve, reject) => {
-    debug('.well-known/dat lookup for name:', name)
-    https.get({
-      host: name,
-      path: '/.well-known/' + recordName,
-      timeout: 2000
-    }, function (res) {
+function fetchWellKnownRecordWithRedirects (recordName, host, path, followRedirects) {
+  return _asyncToGenerator(function * () {
+    let redirectCount = 0
+    while (redirectCount < followRedirects) {
+      const res = yield fetchWellKnownRecord(recordName, host, path)
+      if ([301, 302, 307, 308].includes(res.statusCode)) {
+        if (!'location' in res.headers) {
+          debug('.well-known/' + recordName + ' lookup redirect did not contain destination Location header.')
+          throw new Error('Well record redirected to nowhere')
```

Comment:
Minor typo in this error description

Reply:
I adjust both this and the next error message in 7edfc1b